packages/react-ui-kit/src/Layout/Theme.tsx

```diff
@@ -34,6 +34,7 @@ export interface Theme {
   general: {
     backgroundColor: string;
     color: string;
+    themeColor?: string;
```

Comment:
Unrelated change?

Reply:
That said, I like the idea of having this generic `themeColor` that could be applied across all components :ok_hand: